pkg/platform/packet/packet.go

```diff
@@ -76,6 +88,7 @@ type config struct {
 	ReservationIDs           map[string]string `hcl:"reservation_ids,optional"`
 	ReservationIDsDefault    string            `hcl:"reservation_ids_default,optional"`
 	CertsValidityPeriodHours int               `hcl:"certs_validity_period_hours,optional"`
+	NodesDependsOn           string            // Not exposed to the user
```

Comment:
I wonder if this picks up the "default" label, the same as YAML and JSON, where you need to do explicitly `json:"-"` to skip it from marshaling/unmarshaling.

Reply:
Not sure I follow. What is the issue?

Reply:
```go
package main

import (
  "encoding/json"
  "testing"
)

type F struct {
  Foo string
}

func TestMarshal(t *testing.T) {
  f := F{
    Foo: "bar",
  }

  b, err := json.Marshal(f)
  if err != nil {
    t.Fatalf("failed to encode: %v", err)
  }

  t.Logf("result: %+v", string(b))
}
```

I mean does something like this work for HCL. No major issue here, just a wondering if we should maybe add `hcl:"-"` to not unmarshalable struct fields.

Reply:
But why is this relevant? Do you think we may use it? Or it would be nice if the struct is marshall-able for some reason?

Reply:
I posted incomplete example. The same works for `Marshal` and `Unmarshal`. Anyway..

Reply:
Not sure what possible issue do you refer to, but hcl is loaded just fine :)